### ISSUE TITLE
RubberDuck should allow a user to clone a vbComponent

### ISSUE THREAD

**Issue by ThunderFrame** [rubberduck-vba/Rubberduck#1312]
VBE doesn't offer a way to clone/duplicate a Standard Module, Class or UserForm, other than exporting the component, manually editing the files, and then reimporting.

Rubberduck should automate that process so that a user can clone a component with minimal effort. Ideally, RD asks for the name of the new component, and maybe even makes a default suggestion like "Copy of frmMain"

**Comment by ThunderFrame** [comment]
RubberDuck should also clean up the temp files.

**Comment by retailcoder** [comment]
Basically, add copy+paste to the project (native) and code (ours) explorer toolwindows' context menus. Right? (and ideally honor Ctrl+C/Ctrl+V hotkeys too)

**Comment by ThunderFrame** [comment]
Yep, but I don't think we can clone a document-type VBComponent quite as easily.
